starter-code/css/main.css

```diff
@@ -1,10 +1,133 @@
+/* General */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font: 200 16px/1.5 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  margin: 0 auto;
+  /* box-sizing: border-box; */
+}
+
+h2 {
+  width: 320px;
+  font-size: 30px;
+  color: #143c5e;
+  margin-bottom: 20px;
+}
+
+p {
+  font-size: 20px;
+}
+
+.wrapper {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+/* Blue background */
+
+.blue-back {
+  background-color: #4e7495;
+  height: 400px;
+  width: 100%;
```

Comment:
good use of height but you can reduce the height a bit to make it look more like the site your cloning.